build/yamls/base/crds.yml

```diff
@@ -9,6 +9,10 @@ spec:
     - name: v1beta1
       served: true
       storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          x-kubernetes-preserve-unknown-fields: true
```

Comment:
this ensures original behavior. but imo we should update them to perform strict validation in a follow up for individual CRDs